media/js/firefox/new/experiment-firefox-new-waitface-switch.js

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var isDesktop = !/android|ios/.test(site.platform);
+
+    var notFirefox= !/\sFirefox/.test(navigator.userAgent);
+
+    var paramsMatch = function() {
+        var params = {
+            'utm_medium': 'display',
+            'utm_source': 'facebook',
+            'utm_campaign': 'x',
+            'utm_content': 'switch-lp'
+        };
+
+        for (var key in params) {
+            if (!(new RegExp('[\&\?]' + key + '=' + params[key])).test(location.search)) {
+                return false;
+            }
+        }
+
+        return true;
+    };
+
+    if (isDesktop && notFirefox && paramsMatch()) {
+        var marimow = new Mozilla.TrafficCop({
+            id: 'experiment_firefox_new_waitface_switch',
```

Comment:
Interestingly I noticed that when I have cookies for both the previous experiment and this new one, I keep getting redirected to `utm_content=switch-lp&v=2&v=a`. Looking through the traffic cop & cookie helper code, I believe it's because this `id` gets matched (via a regexp) with the older experiment id `experiment_firefox_new_waitface`. This should probably be fixed in Traffic Cop (?), but for now perhaps we could just use a more unique `id`?